vuepress/docs/v7.1/tutorials/create/ms/generate-microservices-and-micro-frontends.md

```diff
@@ -43,7 +47,7 @@ ent jhipster --blueprints=entando
     - `Which *development* database would you like to use?` (H2 with disk-based persistence)
      - `Which cache do you want to use? (Spring cache abstraction)` (Caffeine (local cache, for a single node))
     - `Do you want to use Hibernate 2nd level cache?` (Yes)
-    - `Which other technologies would you like to use?` (**Don't select any other technologies**)
+    - `Which other technologies would you like to use?` (Don't select any other technologies)
     - `Which BE dependencies do you want to use?` (Dependencies maintained by Entando (entando/entando-bundle-bom))
     - `Would you like to generate micro frontends when creating entities?` (Always)
     - `Would you like to enable internationalization support` (Up to you)
```

Comment:
did you also see that the parenthesis entry for pagination is different than we have currently? i had made a note to update it but wanted to check with you

Reply:
I left it as is, because the parenthesis are there to signify what to answer, not so much as a duplicate of what is printed on the console. I think it's clearer that way. But what entry for pagination?

Reply:
no i agree about the parentheses prompts; no changes suggested. i just meant that the pagination prompt i see reads "Yes, with infinite scroll and sorting headers" - super minor but different from what it used to be